src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -9,13 +9,13 @@
 5
 
 >> Value(ParseJSON("true"))
-Error({Kind:ErrorKind.InvalidArgument})
+1
 
 >> Value(ParseJSON("false"))
-Error({Kind:ErrorKind.InvalidArgument})
+0
 
 >> Value(ParseJSON("""5"""))
-Error({Kind:ErrorKind.InvalidArgument})
+5
```

Comment:
Can we add this test case::

// Text conversion uses PowerFx rules, not Json parsing. 
Value(ParseJSON("""123,456""")) 
123456

Reply:
Added